kf_lib_data_ingest/common/file_retriever.py

```diff
@@ -86,14 +101,30 @@ def _s3_save(protocol, source_loc, dest_obj, auth=None, logger=None):
     raise botocore.exceptions.NoCredentialsError()  # never got the file
 
 
-def _web_save(protocol, source_loc, dest_obj, auth=None, logger=None):
+def _web_save(protocol, source_loc, dest_obj, auth=None, auth_config=None,
```

Comment:
Why do we have two auth fields now? Can we unify them?

Reply:
Yes, addressed 👍